content/en/dashboards/querying/_index.md

```diff
@@ -210,7 +210,17 @@ If your level of detail is resource or span, some widget types also require you
 
 ### Event overlays
 
-View event correlations by using the **Event Overlays** section in the graphing editor. In the search field, enter any text or structured search query. For details on searching, see the Datadog [Event Query Language][14].
+View event correlations by using the **Event Overlays** section in the graphing editor for the [Timeseries][15] visualization. In the search field, enter any text or structured search query. For details on searching, see the Datadog [Event Query Language][14].
```

Comment:
I think the link for Event Query Language is broken... Can you confirm?

Reply:
Ooh, good catch, yea this section doesn't seem to exist